index.js

```diff
@@ -273,13 +275,21 @@ walletRouter.post('/send-tx', (req, res) => {
       const response = {success: false, error};
       if (debug) {
         response.debug = ret.debug;
+        logger.debug('/send-tx failed', {
+          body: req.body,
+          response: response,
+        });
       }
       res.send(response);
     });
   } else {
     const response = {success: false, error: ret.message};
     if (debug) {
```

Comment:
Is `if (debug)` required? I suppose `winston` should automatically ignore some logs based on the `loglevel` defined [here](https://github.com/HathorNetwork/hathor-wallet-headless/pull/54/files#diff-4e4bb3f3b1a497e10bb5d503f3ccb5072a1c0540e0020a00fa20da2cfd3ba2b3R26) and passed to `winston` [here](https://github.com/HathorNetwork/hathor-wallet-headless/pull/54/files#diff-ed4a110a094ac014f4bea8d6d93719d89d5e5a2d1974dc2f79e41faf30af470aR22).

Reply:
This `debug` is related to the HTTP Request (and not to `winston`). It means the transaction is failing and the caller would like to get debugging information to find out what's wrong.